src/modules/vtol_att_control/standard.cpp

```diff
@@ -231,13 +231,15 @@ void Standard::update_transition_state()
 	}
 
 	if (_vtol_schedule.flight_mode == vtol_mode::TRANSITION_TO_FW) {
-		if (_param_vt_psher_rmp_dt.get() <= 0.0f) {
+		if (_param_vt_psher_slew.get() <= 0.0f) {
 			// just set the final target throttle value
 			_pusher_throttle = _param_vt_f_trans_thr.get();
 
-		} else if (_pusher_throttle <= _param_vt_f_trans_thr.get()) {
+		} else if (_pusher_throttle < _param_vt_f_trans_thr.get()) {
 			// ramp up throttle to the target throttle value
-			_pusher_throttle = _param_vt_f_trans_thr.get() * time_since_trans_start / _param_vt_psher_rmp_dt.get();
+			_pusher_throttle += _param_vt_psher_slew.get() * _dt;
```

Comment:
```suggestion
			_pusher_throttle = math::min(_pusher_throttle + _param_vt_psher_slew.get() * _dt, _param_vt_f_trans_thr.get());
		
```
Otherwise it can grow above _param_vt_f_trans_thr.get() (depending on the slew rate setting and _dt).

Reply:
Sorry I wrote it wrong, it should be 
_pusher_throttle = math::min(**_param_vt_f_trans_thr.get()** + _param_vt_psher_slew.get() * _dt, _param_vt_f_trans_thr.get());
, no? Otherwise it will not increase throttle at all during the front transition...